lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -352,8 +333,17 @@ namespace Js
     typedef FinalizableICUObject<UFieldPositionIterator *, ufieldpositer_close> FinalizableUFieldPositionIterator;
     typedef FinalizableICUObject<UCollator *, ucol_close> FinalizableUCollator;
 
+    template<typename ICUFunc>
+    static void RecyclerExecuteICUWithRetry(_In_ ICUFunc func, _In_ Recycler *recycler, _Out_writes_opt_(returnLen) char16 **ret, _Out_ int *returnLen, _In_ int firstTryLength = 32)
+    {
+        bool success = ExecuteICUWithRetry<char16, ICUFunc>([&](int length) {
```

Comment:
>{ [](start = 76, length = 1)

nit: I think we put { for lambdas on the next line like we do for normal functions

Reply:
Ah, right. I'm not sure how I didn't notice that earlier, this function has been around for a little while by now.